jenkins-builds/6.0.UP02-2016.11-mc

```diff
@@ -2,7 +2,7 @@
  Amber-16-2016.11-CrayGNU-2016.11-parallel.eb
  Boost-1.62.0-CrayGNU-2016.11-Python-3.5.2.eb
  Boost-1.61.0-CrayGNU-2016.11-Python-2.7.12.eb
- CDO-1.7.2-CrayGNU-2016.11.eb
+ CDO-1.8.0rc6-CrayGNU-2016.11.eb
```

Comment:
Are you sure about removing the latest stable and replacing with a release candidate?